lib/features/children/generosity_challenge/assignments/day4_timer/pages/day4_timer_screen.dart

```diff
@@ -112,6 +115,11 @@ class _Day4TimerScreenState extends State<Day4TimerScreen> {
     );
   }
 
+  Future<bool> _isDebugApp() async {
```

Comment:
Should we put this into a more generic place? I think I'm seen this method more and more in our code :)

Reply:
Yea, agreed I'll make a separate PR for it!